spec/features/user_views_orders_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.